ci/ansible/all.yml

```diff
@@ -9,10 +9,10 @@ deploy_user: "root"
 # Docker options
 ###################
 
-ovn_db_image: "huikang/ovn-scale-test-ovn"
-ovn_chassis_image: "huikang/ovn-scale-test-ovn"
+ovn_db_image: "ovn-scale-test-ovn"
```

Comment:
I'd prefer if this PR had the changes to the above files separate, and a "git revert" of the commit I did here. Can you do that and then I will merge this?